src/main/scala/ai/privado/model/PrivadoTag.scala

```diff
@@ -87,6 +87,7 @@ object Language extends Enumeration {
 
   val JAVA       = Value("java")
   val JAVASCRIPT = Value("javascript")
+  val PYTHON = Value("python")
   val DEFAULT    = Value("default")
```

Comment:
remove the ImportUtility related implementation, that will go as part of enhanced API branch